app/flatpak-cli-transaction.c

```diff
@@ -334,7 +334,7 @@ progress_changed_cb (FlatpakTransactionProgress *progress,
 
           formatted_max = g_format_size (max);
           if (transferred < 1024) // avoid "bytes"
-            formatted = g_strdup ("1.0 kB");
+            formatted = g_format_size (1024);
```

Comment:
The docs for `g_format_size` say it uses powers of 10 not powers of 2. Perhaps we should change this to 1000 and change the if condition as well.